core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt

```diff
@@ -31,6 +33,20 @@ abstract class ToolkitCredentialsProvider : AwsCredentialsProvider {
     override fun hashCode(): Int = id.hashCode()
 
     override fun toString(): String = "${this::class.simpleName}(id='$id')"
+
+    open fun isValid(sdkHttpClient: SdkHttpClient): Boolean {
```

Comment:
Can we remove the parameter and just use `AwsSdkClient.getInstance().sdkHttpClient`? I don't see the reason for using other SDK client.

Reply:
This is in core, which doesn't have a reference to `jetbrains-core`, where `AwsSdkClient` resides

Reply:
Ah, gotcha